pkg/controllers/common/conditions.go

```diff
@@ -104,3 +105,30 @@ func UpdateControllerConditions(ctx context.Context, operatorClient v1helpers.Op
 
 	return nil
 }
+
+func ApplyControllerConditions(ctx context.Context, operatorClient v1helpers.OperatorClient, fieldManager string, allConditionNames sets.String, updatedConditions []operatorv1.OperatorCondition) error {
```

Comment:
dead?  I think I do prefer the "normal" style over this.

Reply:
Sorry you reviewed the PR at the point :). This was a copy-paste from the function above in order to allow me to gradually migrate controllers. This is gone now.